gear-programs/vft-treasury/src/services/mod.rs

```diff
@@ -169,22 +176,33 @@ where
         }
 
         self.data_mut()
```

Comment:
Please also check there that there're no vara address that maps to multiple eth addresses and no eth address mapped to multiple vara ones